tools/monitoring/alert/alert.go

```diff
@@ -59,6 +59,8 @@ func (c *Client) RunAlerting() {
 }
 
 func (c *Client) handleReportMessage(rmsg *prowapi.ReportMessage) {
+	log.Printf("Received Pubsub message %v\n", rmsg)
```

Comment:
should this be insied if condition? Else, we will be logging for every job every status change.

Reply:
Yes, you're right. Updated.